fvm/env.go

```diff
@@ -94,23 +94,19 @@ func (e *hostEnv) getLogs() []string {
 }
 
 func (e *hostEnv) GetValue(owner, key []byte) ([]byte, error) {
-	v, _ := e.ledger.Get(
-
-		string(owner),
-		"", // TODO: Remove empty controller key
+	v, _ := e.accounts.GetValue(
+		flow.BytesToAddress(owner),
 		string(key),
 	)
```

Comment:
This assumes every register is owned by an account, are we okey with this assumption?

Reply:
Not all registers are owned by addresses, but all `owner`s that go currently through this function are addresses. I think that the owner here was always intended to be an address. The comments even state so

```go
// GetValue gets a value for the given key in the storage, owned by the given account.
GetValue(owner, key []byte) (value []byte, err error)
// SetValue sets a value for the given key in the storage, owned by the given account.
SetValue(owner, key, value []byte) (err error)
```

I wanted to change the interface in cadence to accept an address instead of a byte array in a separate PR (before merging the storage feature branch into master). 

@turbolent do you think there is a problem with `owner` having the type of Address in these two functions?

Reply:
When we started working on the interface we didn’t quite know yet if we would ever have another kind of key, so we used the lowest level type possible, a byte array.

I think it's OK to change this now, though it'll be a breaking change – given that the interface is only implemented/used by Flow code, and no users are using it yet, that should be fine though.